src/main/java/com/commercehub/gradle/plugin/avro/GenerateAvroJavaTask.java

```diff
@@ -39,6 +46,7 @@ public void setEncoding(String encoding) {
 
     @Input
     public String getStringType() {
+        System.out.println("StringType was get: " + stringType);
```

Comment:
Thanks! I added it while trying to debug; I was trying to figure out why the getStringType etc methods seem to have some "magic" to them that I still don't understand. There's one more compiler setting (createSetters) that I'd be glad to implement as well to have a complete feature set. 

Great plugin; we use it extensively here at Under Armour!